example-react-app/src/app/form-wizard-CompositionO2O-editor/FormWizardCompositionO2O.tsx

```diff
@@ -0,0 +1,139 @@
+    $id: String = ""
+    $loadItem: Boolean!
+  ) {
+    scr_FormWizardCompositionO2OTestEntityById(id: $id)
+      @include(if: $loadItem) {
+      id
+      _instanceName
+      name
+    }
+  }
+`;
+
+const UPSERT_SCR_FORMWIZARDCOMPOSITIONO2OTESTENTITY = gql`
+  mutation Upsert_scr_FormWizardCompositionO2OTestEntity(
+    $formWizardCompositionO2OTestEntity: inp_scr_FormWizardCompositionO2OTestEntity!
+  ) {
+    upsert_scr_FormWizardCompositionO2OTestEntity(
+      formWizardCompositionO2OTestEntity: $formWizardCompositionO2OTestEntity
+    ) {
+      id
+    }
+  }
+`;
+
+const FormWizardCompositionO2O = observer(
+  (props: EntityEditorProps<FormWizardCompositionO2OTestEntity>) => {
+    const {
+      onCommit,
+      entityInstance,
+      submitBtnCaption = "common.submit"
+    } = props;
+
+    const [form] = useForm();
+
+    const {
+      executeLoadQuery,
+      loadQueryResult: { loading: queryLoading, error: queryError },
+      upsertMutationResult: { loading: upsertLoading },
+      serverValidationErrors,
+      intl,
+      handleSubmit,
+      handleSubmitFailed,
+      handleCancelBtnClick
+    } = useEntityEditor<FormWizardCompositionO2OTestEntity>({
+      loadQuery: LOAD_SCR_FORMWIZARDCOMPOSITIONO2OTESTENTITY,
+      upsertMutation: UPSERT_SCR_FORMWIZARDCOMPOSITIONO2OTESTENTITY,
+      entityName: ENTITY_NAME,
+      routingPath: ROUTING_PATH,
+      onCommit,
+      entityInstance,
+      useEntityEditorForm: createUseAntdForm(form),
+      useEntityEditorFormValidation: createUseAntdFormValidation(form)
+    });
+
+    if (queryLoading) {
+      return <Spinner />;
+    }
+
+    if (queryError != null) {
+      console.error(queryError);
+      return <RetryDialog onRetry={executeLoadQuery} />;
+    }
+
+    return (
+      <Card className="narrow-layout">
+        <Form
+          onFinish={handleSubmit}
+          onFinishFailed={handleSubmitFailed}
+          layout="vertical"
+          form={form}
+          validateMessages={createAntdFormValidationMessages(intl)}
+        >
+          <Field
+            entityName={ENTITY_NAME}
+            propertyName="name"
+            formItemProps={{
+              style: { marginBottom: "12px" }
+            }}
+          />
+
+          <GlobalErrorsAlert serverValidationErrors={serverValidationErrors} />
+
+          <Form.Item style={{ textAlign: "center" }}>
+            <Button htmlType="button" onClick={handleCancelBtnClick}>
+              <FormattedMessage id="common.cancel" />
+            </Button>
+            <Button
+              type="primary"
+              htmlType="submit"
+              loading={upsertLoading}
+              style={{ marginLeft: "8px" }}
+            >
+              <FormattedMessage id={submitBtnCaption} />
+            </Button>
+          </Form.Item>
+        </Form>
+      </Card>
+    );
+  }
+);
+
+registerEntityEditor({
+  component: FormWizardCompositionO2O,
+  caption: "screen.FormWizardCompositionO2O",
```

Comment:
Submitting the form results in:

![image](https://user-images.githubusercontent.com/8298981/126331208-c92c6649-afeb-4c6c-bce4-7f6511ed4bcf.png)

(even though GraphQL mutation seems to be successful)

Also editor doesn't close (should it?).

Reply:
Only happens in `Form Wizard Editor` screen.

Reply:
Created the issue for it.
https://github.com/Haulmont/jmix-frontend/issues/510